src/py/flwr/cli/run/run.py

```diff
@@ -54,6 +60,14 @@ def run(
             "inside the `pyproject.toml` in order to be properly overriden.",
         ),
     ] = None,
+    follow: Annotated[
+        bool,
+        typer.Option(
+            "--follow/--no-follow",
```

Comment:
Same statement I posted here:

https://github.com/adap/flower/pull/3577/files#r1696506854

Reply:
Resolved in #3577